stlab/concurrency/channel.hpp

```diff
@@ -591,6 +591,7 @@ template <typename Q, typename T, typename R, typename Arg, std::size_t I, typen
 struct shared_process_sender_indexed : public shared_process_sender<Arg> {
     shared_process<Q, T, R, Args...>& _shared_process;
 
+    explicit
```

Comment:
I would prefer if the subject of the changes is not mixed up. This PR is for adding actions.

Reply:
> I would prefer if the subject of the changes is not mixed up. This PR is for adding actions.

You are right. I have the same preference. I do not why this code ended up here. It is from another PR. I think a made a mistake with branches.